presenters/thread.rb

```diff
@@ -48,7 +49,8 @@ def to_hash(
     end
     sorting_key_order = reverse_order ? -1 : 1
     if with_responses
-      if @thread.thread_type.discussion? && resp_skip == 0 && resp_limit.nil?
+      if (@thread.thread_type.discussion? || (@thread.thread_type.question? && merge_question_type_responses)) &&
```

Comment:
what does this check mean now?

Reply:
The objective of this pull request (PR) is to ensure that the API responses for discussion type threads and question type threads are consistent. To achieve this, the check is  to verify if the merge_question_type_responses parameter is present in the API call and if the thread type is a question. If both conditions are met, the API response for question type threads should match that of discussion type threads.